gsi_wdl_tools/workflow_info.py

```diff
@@ -74,11 +75,12 @@ def outputs(self):
             if not description:
                 raise Exception(f"output description is missing for {name}")        
                 
-            vidarr_label = [] 
+            labels = []
                        
             # Check if 'vidarr_label' exists in the output description
+            # TODO: support for other labels
```

Comment:
what do you mean by 'other labels'? labels made the traditional way (with an explicit Pair) are already supported by the code at 83

Reply:
For example, bcl2fastq produces labels https://github.com/oicr-gsi/bcl2fastq/blob/master/bcl2fastq.wdl#L17 but there is no way to report on what these labels are.